recipes/museotoolbox/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "museotoolbox" %}
+{% set version = "0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 30201a51d6033c1b191bd822741f7cbf073053926928d818ea874bfcad3f9917
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - gdal >=2
+    - numpy >=1.7
+    - pip
+    - python
```

Comment:
I think on ``host`` you just need ``pip`` and ``python``

Reply:
@marcelotrevisani , I updated the meta.yaml according to your suggestion : https://github.com/conda-forge/staged-recipes/pull/10863/commits/aadd112f482aa47787a0f9eb529aef6f162e72f9.
Thank you.

Reply:
I think you do not need ``gdal`` in ``host``, just in ``run`

Reply:
Sorry @marcelotrevisani, indeed no need for `gdal` in `host`. It's updated and checked : https://github.com/conda-forge/staged-recipes/pull/10863/commits/e4f4a94091f5aebcb53ba3062ae13e037cc39ff3.